### ISSUE TITLE
how do I know the loss is average among devices when using ddp

### ISSUE THREAD

**Issue by zixiliuUSC** [PyTorchLightning/pytorch-lightning#12407]
## 🐛 Bug

As lots of issues said, the loss reported in progress bar is not correct but summing among different devices. I am confused whether the loss is not averaged before backward. How do I check this?

**Comment by rohitgr7** [comment]
hey @zixiliuUSC !
loss shown on the progress bar is only from the device with global_rank = 0 and is not averaged. The gradients are averaged (synced) instead during `optimizer.step`.
